README.md

```diff
@@ -35,7 +35,7 @@ This section provides a summary of the ways different types of content can be ad
 
 ### Creating a team member profile
 1. Add your details in `/data/people.yml` file
-2. Put a little photo of yourself in `/assets/img/team`. Filename should match the `pic` value you provided with your details, above
+2. Put a little photo of yourself in `/assets/img/team`. Filename should match the `pic` value you provided with your details above. For how to take a profile photo, see our [Team Profile Photo Guide](https://docs.google.com/presentation/d/1JeL-FCusWLBF71zpd419F1_Tr2rK8vAoT4EHqSvPg2c/edit)
```

Comment:
Maybe we can add this as the first step?

Something like:

### Creating a team member profile
0. Take a profile photo. Visit our [Team Profile Photo Guide](https://docs.google.com/presentation/d/1JeL-FCusWLBF71zpd419F1_Tr2rK8vAoT4EHqSvPg2c/edit) to see how.
1. Add your details...

Reply:
And then we can update step 2 to say something like:
2. Ensure your photo is of the same dimensions as the others and put it in the `/assets/img/team` folder. Etc Etc

Reply:
Thanks for the feedback Shaii - changes applied.